src/main/java/com/github/martynagil/drugstoremanagement/service/EmployeeService.java

```diff
@@ -39,4 +47,14 @@ public void dismissEmployee(Long employeeId, EmployeeDismissalDto employeeDismis
                 .orElseThrow(() -> new EntityNotFoundException());
         employee.dismiss(employeeDismissalDto.getDismissalDate());
     }
+
+    private Boolean employeeExists(EmployeeDto employeeDto) {
```

Comment:
To sprawdzanie jest bez sensu i w zasadzie przed niczym nie chroni. Pominąłbym je całkowicie